color/src/main/res/layout-land/md_color_custom_color_chooser.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/llCustomColor"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:paddingLeft="@dimen/color_grid_item_padding_double"
+    android:paddingRight="@dimen/color_grid_item_padding_double"
+    android:paddingBottom="@dimen/color_grid_padding_bottom">
+
+    <FrameLayout
+        android:layout_width="128dp"
+        android:layout_height="128dp"
+        android:layout_marginRight="16dp"
+        android:background="@drawable/transparent_rect_repeat">
+
+        <View
+            android:id="@+id/v_color"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+    </FrameLayout>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:id="@+id/llAlpha"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <TextView
+                android:id="@+id/tv_alpha"
+                style="@style/TextAppearance.AppCompat.Medium"
+                tools:ignore="HardcodedText"
```

Comment:
I think you'd want to put this ignore on the root layout so it applies to all of the other views in this layout which are using hardcoded text